mesonbuild/backend/ninjabackend.py

```diff
@@ -1606,7 +1606,7 @@ def generate_compile_rule_for(self, langname, compiler, is_cross, outfile):
             output_args=' '.join(compiler.get_output_args('$out')),
             compile_only_args=' '.join(compiler.get_compile_only_args())
         )
-        description = ' description = Compiling %s object $out.\n' % langname.title()
+        description = ' description = Compiling %s object $out.\n' % compiler.get_display_language()
```

Comment:
That does not work with `C#`, but I think it's okay since it has its own rule generator anyway.